src/model/fsm_util.jl

```diff
@@ -27,7 +27,7 @@ Returns:
 function load_bvn_mixtures{NumType <: Number}(
```

Comment:
Ok --- i updated the comment above `load_bvn_mixtures` to account for `psf_K`.  I pattern matched on this refactor -- I'm not super sure what all of the inputs from `ElboArgs` were.